djautotask/api.py

```diff
@@ -930,3 +930,45 @@ def delete(self, parent, **kwargs):
             kwargs.get("id")
         )
         return self.request('delete', endpoint_url)
+
+class SystemAPIClient(AutotaskAPIClient):
```

Comment:
The name `SystemAPIClient` comes from CW so doesn't really make sense to use here. Instead lets call this `Attachment` or `AttachmentInfo` `APIClient`

Reply:
Resolved.